server/models/notes.js

```diff
@@ -0,0 +1,32 @@
+const mongoDB = require('../config/mongoDB')
+
+const note = mongoDB.Schema({
+    slug:{
+        type: String,
+        required: [true, "smalid is required"]
```

Comment:
Let's install the package from npm and use its generate method as default here, so that we don't need to add it everytime we create a new note. The `shortid` should be autogenerated on create.